tools/anthosbm-ansible-module/inventory/staging-hybrid/group_vars/all/000_anthos_common_var.yml

```diff
@@ -0,0 +1 @@
+../../../000_anthos_common_var.yml
```

Comment:
Is the ansible-playbook command abled to use this group_vars file? The ansible documentation says 
"You can also add group_vars/ and host_vars/ directories to your playbook directory. The ansible-playbook command looks for these directories in the current working directory by default. Other Ansible commands (for example, ansible, ansible-console, and so on) will only look for group_vars/ and host_vars/ in the inventory directory."
Source: https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html

Reply:
Yes ansible-playbook is able to use this group_vars file

Reply:
Then we are fine. Thanks for clarifying.